library/dd-sdk-android-rum/src/test/kotlin/com/datadog/android/rum/utils/assertj/DeserializedMapAssert.kt

```diff
@@ -0,0 +1,59 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.rum.utils.assertj
+
+import com.datadog.android.rum.utils.assertj.JsonElementAssert.Companion.assertThat
+import com.google.gson.JsonElement
+import org.assertj.core.api.AbstractAssert
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.fail
+import java.util.Date
+
+// TODO RUMM-2949 Share forgeries/test configurations between modules
```

Comment:
Yes we really need to do this soon, we have too many duplicates in the testing sources. In SR I have the same problem.